src/prof_log.c

```diff
@@ -623,11 +624,19 @@ prof_log_stop(tsdn_t *tsdn) {
 		return true;
 	}
 
-	/* Emit to json. */
 	struct prof_emitter_cb_arg_s arg;
 	arg.fd = fd;
-	emitter_init(&emitter, emitter_output_json, &prof_emitter_write_cb,
-	    (void *)(&arg));
+
+	char *prof_log_stop_buf = (char *)iallocztm(tsdn,
+	    PROF_LOG_STOP_BUFSIZE * sizeof(char),
```

Comment:
No need to do the sizeof(char), it's always 1.

Also, do we have similar pattern for the current prof dump buffer? If so, let's also do a similar optimization there.